### ISSUE TITLE
bug: authors are required

### ISSUE THREAD

**Issue by bsureshkrishna** [m2b3/SciCommons-frontend#82]
![image](https://github.com/user-attachments/assets/7e7fd648-9395-4a9d-adc9-2cce4931e897)

**Comment by bsureshkrishna** [comment]
added a tag that looks like an author, and now get the error, An error occurred (403) when calling the HeadObject operation: Forbidden

This is when trying to create a hidden community

**Comment by dinakar17** [comment]
The problem is associated with how AWS is set up by Armaan. It requires a slight modification in the person's account whose AWS credentials are being used.
https://stackoverflow.com/questions/36144757/aws-cli-s3-a-client-error-403-occurred-when-calling-the-headobject-operation

**Comment by bsureshkrishna** [comment]
@dinakar17 there is also an"author required" issue

**Comment by rajubugude** [comment]
I couldn't see this bug now. please confrim @bsureshkrishna , we can close this.

**Comment by bsureshkrishna** [comment]
yes, it is fixed. but it would be good to add a line -- so if i enter a tag and then do not hit enter, the text is not converetd into a tag, and whne one clicks create, the error is "tags are required". would be good to say, type text and hit enter to create a tag
